src/App.php

```diff
@@ -295,31 +295,6 @@ public function caughtException(Throwable $exception)
         return true;
     }
 
-    /**
-     * Most of the ajax request will require sending exception in json
-     * instead of html, except for tab.
-     *
-     * @return bool
-     */
-    protected function isJsonRequest()
```

Comment:
Even though this function would result in a one-liner, I like having this function. Typing `$app->isJsonRequest()` seems better than having to put `isset($_GET['__atk_json'])` everywhere. Think of IDE hinting :)

Reply:
see also #978, WDYT?

Reply:
I removed it intentionally as it might not be valid all the times, i.e. when the current URL was not made by `getJSURL`.

But the more I worked on this PR, the more I understand the tight integration of HTML/JS URLs/requests in `App`. So to make everyone happy I will restore it, rename it to `isJsRequest` and make it public. Win-win?